### ISSUE TITLE
Different setting of MAX_ATTEMPT attribute in different background task

### ISSUE THREAD

**Issue by knm1993** [arteria/django-background-tasks#256]
I notice that we can have a [global setting of max_attempt](https://django-background-tasks.readthedocs.io/en/latest/#settings) in the settings.py.  
However, can the background task has its own max attempt number? Since some of the background task are supposed to abort on fail.

Would there be any setting in decorator like this?
```
@background(schedule=10, max_attempt=1)
def foo(bar):
     #blah blah blah
```

**Comment by LefterisJP** [comment]
I also have the same question. Would there be a way to repeat some tasks, but abort others when they fail?

**Comment by rjsnh1522** [comment]
I am also looking for solution for this?? Have u found and solution to this??

**Comment by fuzzylogic2000** [comment]
Yes, that is a feature that would also help me. Are there any plans if it can be implemented?
